pkgs/applications/misc/tvbrowser/default.nix

```diff
@@ -0,0 +1,87 @@
+{ lib
+, fetchurl
+, stdenv
+, fetchzip
+, ant
+, jdk
+, makeWrapper
+, callPackage
+}:
+
+let
+  minimalJavaVersion = "11";
+
+  newsPlugin = fetchurl {
+    url = "https://www.tvbrowser.org/data/uploads/1372016422809_543/NewsPlugin.jar";
+    hash = "sha256-5XoypuMd2AFBE2SJ6EdECuvq6D81HLLuu9UoA9kcKAM=";
+  };
+in
+assert lib.versionAtLeast jdk.version minimalJavaVersion;
```

Comment:
This assert is not useful. jdk should be at least that minimal version and if people change it via an overlay, then they need to test that and we cannot consider every possible overlay.